dashboard/test/models/level_group_test.rb

```diff
@@ -154,4 +168,37 @@ def get_external_dsl(id)
     end
   end
 
+  test 'is_plc_evaluation?' do
+    evaluation_multi_1 = EvaluationMulti.create_from_level_builder({}, {dsl_text: get_evaluation_multi_dsl(1)})
+    evaluation_multi_2 = EvaluationMulti.create_from_level_builder({}, {dsl_text: get_evaluation_multi_dsl(2)})
+    non_evaluation_multi = Multi.create_from_level_builder({}, {dsl_text: get_multi_dsl(3)})
+
+    evaluation_level_group_dsl = "
+    name 'plc assessment'
+    title 'plc assessment'
+    submittable 'true'
+
+    page
+    level '#{evaluation_multi_1.name}'
+    level '#{evaluation_multi_2.name}'
+    "
+
+    plc_evaluation = LevelGroup.create_from_level_builder({}, {name: 'evaluation_multi', dsl_text: evaluation_level_group_dsl})
+
+    non_evaluation_level_group_dsl = "
+    name 'non plc assessment'
+    title 'non plc assessment'
+    submittable 'true'
+
+    page
+    level '#{evaluation_multi_1.name}'
+    level '#{non_evaluation_multi.name}'
+    "
+
+    non_plc_evaluation = LevelGroup.create_from_level_builder({}, {name: 'evaluation_multi', dsl_text: non_evaluation_level_group_dsl})
```

Comment:
weird, it doesn't cause problems to have these two levels have the same name? (maybe it doesn't because it is not actually writing files?)